lib/scorecard_api/configuration.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'common/client/middleware/response/scorecard_api_errors'
+require 'common/client/middleware/response/json_parser'
+require 'common/client/middleware/response/raise_error'
+require 'common/client/middleware/response/snakecase'
```

Comment:
You have to `require` these because the middlewares have to be registered with Faraday before they can be used.
Those register statements are in the middlewares themselves.  e.g.
> Faraday::Response.register_middleware scorecard_api_errors: Common::Client::Middleware::Response::ScorecardApiErrors

Alternatively, you could put all of those registration statements in an initializer like [this](https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/initializers/faraday_middleware.rb).  Then you won't need the `require` statements in files where you use `Faraday`

I don't have a strong opinion. Just wanted to mention it. `vets-api` does both, which seems like a confusing way to go.

Reply:
Added initializer